core/blockchain_impl.go

```diff
@@ -3186,7 +3186,13 @@ func (bc *BlockChainImpl) isInitTiKV() bool {
 func (bc *BlockChainImpl) tikvPreemptMaster(fromBlock, toBlock uint64) bool {
 	for {
 		// preempt master
-		if ok, _ := bc.redisPreempt.TryLock(60); ok {
+		lockType, _ := bc.redisPreempt.TryLock(60)
```

Comment:
why 60?  can it be a random number in a certain range?

Reply:
This is the upper limit time for writing a block normally. The definition of 60 is because I think it will be more reasonable, writing a block usually takes less than 60 seconds, in case of accident, other writers will take over writing the block after 60 seconds